gptcache/utils/__init__.py

```diff
@@ -258,7 +258,7 @@ def import_fastapi():
 
 def import_redis():
     _check_library("redis")
-    _check_library("redis_om")
+    _check_library("redis-om")
```

Comment:
_check_library("redis_om", package="redis-om")

Reply:
@SimFG Got it thanks. Please check.